docs/GettingStarted.md

```diff
@@ -244,8 +244,8 @@ Install and configure the Datadog Agent to receive traces from your now instrume
 
     ```ruby
     require 'opentracing'
-    require 'ddtrace'
-    require 'ddtrace/opentracer'
+    require 'datadog/tracing'
```

Comment:
About this change -- are we going to start recommending that customers `require` individual products of ddtrace vs just doing `require 'ddtrace'` (or `require 'datadog'`?)

Reply:
I'm leaning towards leaving `require 'ddtrace'` for now, just to avoid one more breaking change.
Also, it aligns with "I installed the gem 'ddtrace', I should be able to 'require "ddtrace"'."

Until we actually have polished our individual product support, I'm hesitant to recommend partial require statements to users.

Regarding `require 'datadog'`, I believe we should wait until our master-gem is officially the de facto Datadog gem, and not today's `ddtrace`.

Reply:
`require 'ddtrace'` will still work fine; `datadog/tracing` in theory becomes a way to individually include tracing, but not the "normal" way to do it right now.